pkg/install/install.go

```diff
@@ -63,3 +72,18 @@ func FillInTemplates(params TemplateParameters) (map[string][]byte, error) {
 	}
 	return result, nil
 }
+
+func ConfigContent(configFile io.Reader, configAsConfigMap bool) (string, error) {
+	tmp, err := ioutil.ReadAll(configFile)
+	if err != nil {
+		return "", fmt.Errorf("unable to read config file: %s", err)
+	}
+	var f chartutil.Files
```

Comment:
Having helm as a dependency just for this seems overkill.

Reply:
When the original PR was made, Helm was already a dependency. The requirement is more modest now, though; this can just be done in a template, as you suggest elsewhere.